docs/use-cases/mobile-driving-license/mdl-architecture/eudi-wallet.md

```diff
@@ -0,0 +1,4 @@
+# EUDI Wallet
+According to Article 5.2 of 4th Driving License Directive mDL shall be issued to the European Digital Identity Wallets as electronic attestations of attributes. Therefore, mDL can only be issued to an active EUDI Wallet instance. 
```

Comment:
Could we have a reference/link to "Article 5.2 of 4th Driving License Directive mDL"